README.md

```diff
@@ -1126,6 +1127,35 @@ Tests whether any of the elements in the array pass the test implemented by the
 
 **[⬆ back to top](#quick-links)**
 
+### _.sortBy
+
+Sorts an array of object based on an object key provided by a parameter (note this is more limited than Underscore/Lodash).
+
+  ```js
+  const fruits = [
+    {name:"banana", amount: 2},
+    {name:"apple", amount: 4},
+    {name:"pineapple", amount: 2}
+    {name:"mango", amount: 1},
+  ];
+
+  // Underscore
+  _.sortBy(fruits, 'name');
+  // => [{name:"apple", amount: 4}, {name:"banana", amount: 2}, {name:"mango", amount: 1}, {name:"pineapple", amount: 2}]
+
+  // Lodash
+  _.orderBy(fruits, ['name'],['asc']);
+  // => [{name:"apple", amount: 4}, {name:"banana", amount: 2}, {name:"mango", amount: 1}, {name:"pineapple", amount: 2}]
+
+  // Native
+  let sortBy = (key) => {
```

Comment:
maybe use `const` is better here since sortBy never changed.

Reply:
Yes definitely makes more sense.